src/mantine-dates/src/components/TimeInput/TimeInput.tsx

```diff
@@ -45,6 +49,8 @@ export const TimeInput = factory<TimeInputFactory>((_props, ref) => {
       step={withSeconds ? 1 : 60}
       type="time"
       __staticSelector="TimeInput"
+      min={props.minTime}
```

Comment:
There is no reason to define custom props for this feature as you just pass it to the input wihout any modifications. Please remove `minTime` and `maxTime` props. In DatePickerInput component use `min` and `max` props on the TimeInput instead

Reply:
got it